javascript/javascript1/week3/distance.js

```diff
@@ -0,0 +1,16 @@
+//When will we be there??
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  
+  function calTime(travelInformation)
+  {
+      let time = (travelInformation.destinationDistance)/(travelInformation.speed);
+      let hour = Math.trunc(time);
+      let min = (travelInformation.destinationDistance)%(travelInformation.speed);
+      return {hour, min};
```

Comment:
Nice detail returning an object with the `hour` and `min` :+1: 

Though the homework wanted you to return a formatted string (what you did outside the function).

IMO leave you solution because it is nice, but know that it was not what was requested :wink:

Reply:
ok..